manual/mapper/daos/select/README.md

```diff
@@ -51,7 +51,7 @@ public interface ProductDao {
 }
 ```
 
-If the annotation has a `customWhereClause`, it completely replaces the WHERE clause. The provided
+If the annotation has a [customWhereClause()], it completely replaces the WHERE clause. The provided
```

Comment:
1. This method has already been referenced line 16 above, I think the link would be more appropriate there than here.
2. Nit: you can use the following syntax to preserve monospace formatting inside links: 

```
[`customWhereClause()`][customWhereClause()]
```

Reply:
1. will do
2. I tend to avoid other formatting when the text is already a link, monospace in particular doesn't render very well IMO.